packages/components/.storybook/addons/bug-hint/Tool.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export const Tool = () => {
+  return (
+    <div
+      style={{
+        color: 'black',
+        display: 'flex',
+        alignItems: 'center',
+        fontSize: '14px',
+        userSelect: 'none'
+      }}
+    >
+      <span>
+        Did you find a bug? Please check the{' '}
+        <a href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-known-browser-issues--docs">
```

Comment:
Please setup some styling for the link, that the color fits to the main color we have in Storybook.

Reply:
I added the primary color for the anchor. But it seems that adding pseudo-classes (eg: :visited, :active) are not super straightforward in React. @mariohamann Should I invest the time to try to implement them too, or is the current update enough?

Reply:
No. It's enough.